src/apps/seelenweg/styles/global.css

```diff
@@ -20,22 +20,59 @@ body {
   height: min-content;
   width: min-content;
 
+  &:has(.taskbar.hidden)::after {
+    content: '';
+    position: fixed;
+    bottom: 0;

```

Comment:
remove unused css code, position an sizes are declared by position so this default values are unnecessary
```css
    bottom: 0;
    left: 0;
    width: 100vw;
    height: 5px;
```

Reply:
there was a msitake in it so it was good you made a comment (on the right i forgot to try...), but for the comment: this is kind of special cause of fixed position, these are not default. 

static position has to tell what is to stick point: 
![image](https://github.com/user-attachments/assets/08c6d9b1-db2a-498d-a4e4-115dc7f7d3c2)

without it: 
![image](https://github.com/user-attachments/assets/22c803ac-594f-4a5b-8e24-93a8b6a11ca4)

Reply:
I refer top, left, right, bottom properties are declared by side, same for width and height.
```css
&:has(.taskbar.hidden.left) {
 &::after {
 ... here are declared
```

Reply:
I see you remove the positions but height and width are duped too, remove them.

Reply:
Eh, sorry, you are right.